pygsti/objects/operation.py

```diff
@@ -2656,11 +2720,15 @@ class StochasticNoiseOp(LinearOperator):
         if not None, a list of `dim-1` initial error rates along each of
         the directions corresponding to each basis element.  If None,
         then all initial rates are zero.
+    
+    seed_or_state : float or RandomState, optional
+        Random seed for RandomState (or directly provided RandomState)
+        for sampling stochastic superoperators with the 'chp' evotype.
```

Comment:
Maybe say "when using a WeakForwardSimulator" rather than "with the 'chp' evotype"?

Reply:
I originally agreed with this, but taking a closer look, this restriction is actually about the representation, not the type of forward simulation. The problem is that CHP only has C, H, and P as native gates, so we have to sample the stochastic map. For example, WeakForwardSimulators with a densitymx evotype could use the usual StochasticNoiseOp representation and therefore not use the seed_or_state kwarg.

Let me know if you still think I should change it.

Reply:
All these changes/comments look good.